crates/hc/src/save.rs

```diff
@@ -96,3 +97,65 @@ pub fn list(hc_dir: PathBuf, verbose: usize) -> anyhow::Result<()> {
     msg!("{}", out);
     Ok(())
 }
+
+lazy_static::lazy_static! {
+    static ref FILE_LOCKS: tokio::sync::Mutex<Vec<usize>> = tokio::sync::Mutex::new(Vec::new());
+}
+
+/// Lock this setup as running live and advertise the port
+pub async fn save_port(mut hc_dir: PathBuf, path: &Path, port: u16) -> anyhow::Result<()> {
```

Comment:
Docsstring doesn't match name at all. Either this function "saves ports" or it "locks this setup".